package.json

```diff
@@ -1,48 +1,45 @@
-{
-  "name": "@nodesecure/size-satisfies",
-  "version": "1.1.0",
-  "description": "Same as SemVer.satisfies but for file size!",
-  "exports": "./index.js",
-  "scripts": {
-    "lint": "cross-env eslint index.js",
-    "test-only": "cross-env esm-tape-runner 'test/**/*.js' | tap-monkey",
-    "test": "npm run lint && npm run test-only"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/NodeSecure/size-satisfies.git"
-  },
-  "files": [
-    "index.js",
-    "index.d.ts"
-  ],
-  "keywords": [
-    "size",
-    "satisfies",
-    "utility",
-    "parse",
-    "parser",
-    "convert",
-    "converter"
-  ],
-  "engines": {
-    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
-  },
-  "author": "GENTILHOMME Thomas <gentilhomme.thomas@gmail.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/NodeSecure/size-satisfies/issues"
-  },
-  "homepage": "https://github.com/NodeSecure/size-satisfies#readme",
-  "dependencies": {
-    "bytes": "^3.1.2"
-  },
-  "devDependencies": {
-    "@nodesecure/eslint-config": "^1.6.0",
-    "@small-tech/esm-tape-runner": "^2.0.0",
-    "@small-tech/tap-monkey": "^1.4.0",
-    "cross-env": "^7.0.3",
-    "tape": "^5.6.3"
-  },
-  "type": "module"
-}
+{
+  "name": "@nodesecure/size-satisfies",
+  "version": "1.1.0",
+  "description": "Same as SemVer.satisfies but for file size!",
+  "exports": "./index.js",
+  "scripts": {
+    "lint": "cross-env eslint index.js",
```

Comment:
Please use `eslint index.js` (you can remove cross-env from the dependency list).